sale_last_price_per_partner/test/__init__.py

```diff
@@ -0,0 +1,5 @@
+# -*- coding: utf-8 -*-
+# Copyright 2017 Camptocamp
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from . import test_last_sale_price_per_partner
```

Comment:
Yes, lint check ignores `__init__.py` files, but it is a good practice to add a newline in the files.